event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventstream/EventStreamJdbcRepository.java

```diff
@@ -137,66 +140,29 @@ private boolean isExistingStream(final UUID streamId) {
         }
     }
 
-    public boolean recordExists(final long sequenceNumber) {
-        try (final PreparedStatementWrapper ps = eventStreamJdbcRepositoryHelper.preparedStatementWrapperOf(dataSource, SQL_RECORD_EXIST)) {
-            ps.setLong(1, sequenceNumber);
-            try (ResultSet rs = ps.executeQuery()) {
-                rs.next();
-
-                return rs.getInt(1) > 0;
+    /**
+     * Returns the latest sequence number.
+     *
+     * @return latest sequence streamId for the stream.  Returns 0 if stream doesn't exist.
```

Comment:
is getCurrentSequence correct?
what does current mean?

Reply:
changed to getLatestSequenceNumber